posthog/api/team.py

```diff
@@ -33,41 +33,68 @@ def has_permission(self, request: request.Request, view) -> bool:
         return True
 
 
+class TeamBasicSerializer(serializers.ModelSerializer):
```

Comment:
Along with `OrganizationBasicSerializer` could consider moving these into own file. It's NIT for now but could see how it becomes unintuitive if there are multiple serializers and some are in shared (user) and some aren't

Reply:
Agreed! Moved all the `**BasicSerializer` to `shared.py`.